lib/ui/window.dart

```diff
@@ -841,6 +841,11 @@ class Window {
   void _respondToPlatformMessage(int responseId, ByteData data)
       native 'Window_respondToPlatformMessage';
 
+  /// Return the version information for the engine, skia and dart.
+  String getFlutterEngineVersion() native 'Window_GetFlutterEngineVersion';
```

Comment:
I think this could be simplified by merging these into one private native method that returns a list of version strings.  Then wrap that with a public Dart method that unpacks the list into a Dart object with fields for each version.

Reply:
That sounds better, will do it.